documentation/ansible-playbook/modules/ansible-playbook-create-bonded-interfaces-using-script.adoc

```diff
@@ -0,0 +1,128 @@
+[id="ansible-playbook-create-bonded-interfaces-using-script.adoc"]
+
+= Create bonded interfaces
+
+If you need to create a bonded provisioning/baremetal interface, the ifcfg files
+can be placed under the "fake" roots for the masters and workers under the
+`roles/installer/files/customize_filesystem/{master,worker}` directories as shown in the above section 4.7
+
+This can also be done by creating a script /usr/local/bin/create-ifcfg-files.sh under 
+`roles/installer/files/customize_filesystem/{master,worker}`
+and by setting by setting variable create_bond_service="true" to create ifcfg files.
```

Comment:
here `by setting` is repeated twice, maybe a typo?

Reply:
done